netbox/dcim/tests/test_models.py

```diff
@@ -554,17 +559,17 @@ def test_connection_via_single_rear_port(self):
         Test a connection which passes through a single front/rear port pair.
 
                      1               2
-        [Device 1] ----- [Panel 1] ----- [Device 2]
+        [Device 1] ----- [Panel 5] ----- [Device 2]
```

Comment:
This test needs to stay the way it is to ensure we test against a multi-position rear port; a new test is needed for the topology with a single-position rear port. I'd rename the current test to `test_connection_via_multi_position_rear_port` and name the new one `test_connection_via_single_position_rear_port`.

I think we're actually missing this condition in the cable validation logic: connecting an interface to a multi-position port probably should not be allowed. (The validation currently only checks if both terminations are rear ports.) This hits on the same conceptual question as #4519: How to treat multiple "A side" terminations mapped to a single "B side" termination.

Reply:
I restored the original `test_connection_via_single_rear_port` code and added a TODO item for when #4604 is going to be merged.